### ISSUE TITLE
frida-tools 12.4.0 distribution fails to run frida-ps and frida-trace

### ISSUE THREAD

**Issue by ishchenko** [frida/frida-tools#157]
`pip install frida-tools` installs frida-16.2.2 and frida-tools-12.4.0

This version has some problems:

1. frida-ps
Running `frida-ps` outputs `"ModuleNotFoundError: No module named '_frida'`. Replacing `import _frida` in ps.py to `import frida._frida as _frida` fixes the issue

2. frida-trace
Running `frida-trace -U -p 12345 -i "Java_*"` outputs `Failed to start tracing: [Errno 2] No such file or directory: '/Users/max/test/venv/lib/python3.12/site-packages/frida_tools/tracer_agent.js'` due to no tracer_agent.js in distribution. But it is present it frida-tools 12.3.0

**Comment by oleavr** [comment]
Whoah, thanks for reporting this so quickly. Just released 12.4.1 with the fix included. Cheers!

**Comment by ishchenko** [comment]
I was lucky enough to install broken packages minutes after release, pure coincidence tbh. Just setting up some ctf environment, and bang, weird ungoogleable stuff happens on my friday evening, but ends happily, yaaaay

**Comment by oldkingOK** [comment]
Hi @oleavr ,

Windows with frida-16.2.2 and frida-tools-12.4.1 by `pip install frida frida-tools` still has the problem.

```
Failed to start tracing: [Errno 2] No such file or directory: 'C:\\Users\\old\\AppData\\Roaming\\Python\\Python310\\site-packages\\frida_tools\\tracer_agent.js'
```

**Comment by oleavr** [comment]
> Hi @oleavr ,
> 
> 
> 
> Windows with frida-16.2.2 and frida-tools-12.4.1 by `pip install frida frida-tools` still has the problem.
> 
> 
> 
> ```
> 
> Failed to start tracing: [Errno 2] No such file or directory: 'C:\\Users\\old\\AppData\\Roaming\\Python\\Python310\\site-packages\\frida_tools\\tracer_agent.js'
> 
> ```

Ooh, so sorry, rushed the fix before bedtime last night and screwed up badly. Just released 12.4.2 with the missing part of the fix. Thanks a lot for reporting!
